src/Signature.cc

```diff
@@ -42,7 +42,7 @@ SectionType snowcrash::SectionKeywordSignature(const mdp::MarkdownNodeIterator&
     TYPECHECK(Relation)
 
     /*
-     *  NOTE: Order is important. Resource MUST preceed the Action.
+     *  NOTE: Order is important. Resource MUST preced the Action.
```

Comment:
This one was correct before. [preceed](http://definefor.me/preceed) is a valid word.